### PR TITLE
Show previous logs

### PR DESCRIPTION
**Pull request by bananaoomarang**
+ Show last 10 lines of logs by default
+ No. of lines to show specifiable with `logs --lines/-n`

Fixes https://github.com/auth0/wt-cli/issues/6

### CODE REVIEW

File: bin/logs.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ module.exports = Cli.createCommand('logs', 'Streaming, real-time logs', {
             'default': 'default',
             type: 'string',
         },
+        lines: {
```
Comment (ggoodman) [inline bin/logs.js]: I think I would prefer something like `history` (but not sure if that even communicates properly). To me, `lines` suggest that this is **all** you will get.

Comment (bananaoomarang) [inline bin/logs.js]: I wasn't sure so I just stole the name that `journalctl` uses (though systemd is hardly a paragon of usability...)

`previous` or `history` might work?

**Comment by ggoodman** [comment]
Did we get the necessary server-side stuff merged for this?

**Comment by bananaoomarang** [comment]
Server side stuff not yet merged, but fixed to make it all work again this morning: https://github.com/auth0/auth0-sandbox-proxy/pull/4

**Pull request closed without merging**